cmd/root.go

```diff
@@ -325,7 +333,7 @@ For more help, see the documentation here: %s`, short, config.Data.Links.CLIDocs
 }
 
 type helpCmd struct {
-	cmd *cobra.Command
```

Comment:
I removed it because `cmd` is passed as an argument of `helpTemplate`, and this `cmd` is not used.